test/keysim_test.js

```diff
@@ -1,15 +1,20 @@
-require("es5-shim");
+import "es5-shim";
 
-var assert = require('assert');
-var Keyboard = require('../dist/keysim').Keyboard;
-var Keystroke = require('../dist/keysim').Keystroke;
-var KeyEvents = require('../dist/keysim').KeyEvents;
+import assert from 'assert';
 
-var isInNode = require('detect-node');
+// whether we test the source or the dist
+import * as src from '..';
+import * as dist from '../dist/keysim';
+const keysim = process.env.DIST ? dist : src;
+const Keyboard = keysim.Keyboard;
+const Keystroke = keysim.Keystroke;
+const KeyEvents = keysim.KeyEvents;
+
+import isInNode from 'detect-node';
 
 // jsdom is required when running in node. Browsers have real DOM.
-var jsdom = isInNode ? require('jsdom') : null;
```

Comment:
I figured out that in this case there's no real need to avoid `require`ing this in the browser. Instead, we use browserify's `ignore` option so that this `require` returns a blank object.